frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -56,38 +56,33 @@ type SegmentInfo = {
 };
 
 const exportStatisticsToCSV = (
-  segmentInformation: Array<SegmentInfo> | string,
+  segmentInformation: Array<SegmentInfo> | null,
   tracingId: string,
   groupIdToExport: number,
   hasAdditionalCoords: boolean,
 ) => {
-  if (segmentInformation.length === 0 || segmentInformation instanceof String) {
-    return;
-  }
   const segmentStatisticsAsString = (segmentInformation as Array<SegmentInfo>)
-    .map(
-      (row) =>
-        [
-          row.additionalCoordinates,
-          row.segmentId,
-          row.segmentName,
-          row.groupId,
-          row.groupName,
-          row.volumeInVoxel,
-          row.volumeInNm3,
-          ...row.boundingBoxTopLeft,
-          ...row.boundingBoxPosition,
-        ]
-          .filter((el) => el != null && el.toString().length > 0)
-          .map(String) // convert every value to String
-          .map((v) => v.replaceAll('"', '""')) // escape double quotes
-          .map((v) => (v.includes(",") || v.includes('"') ? `"${v}"` : v)) // quote it if necessary
-          .join(","), // comma-separated
-    )
+    .map((row) => {
+      const maybeAdditionalCoords = hasAdditionalCoords ? [row.additionalCoordinates] : [];
+      const segmentInfoRow = [
+        row.segmentId,
+        row.segmentName,
+        row.groupId,
+        row.groupName,
+        row.volumeInVoxel,
+        row.volumeInNm3,
+        ...row.boundingBoxTopLeft,
+        ...row.boundingBoxPosition,
+      ]
+        .map(String) // convert every value to String
+        .map((v) => v.replaceAll('"', '""')) // escape double quotes
+        .map((v) => (v.includes(",") || v.includes('"') ? `"${v}"` : v)); // quote it if necessary
+      return maybeAdditionalCoords.concat(segmentInfoRow).join(",");
```

Comment:
instead of `concat`, you can simply use `...maybeAdditionalCoords` within the definition of `segmentInfoRow`.